seed/static/seed/js/controllers/building_detail_controller.js

```diff
@@ -37,7 +38,7 @@ angular.module('BE.seed.controller.building_detail', ['ngToast'])
   'label_helper_service',
   'default_columns',
   function($scope, $routeParams, $uibModal, $log, building_services, project_service, building_payload,
-           building_finer_energy_payload, building_monthly_payload, gb_req_info, all_columns,
+           building_finer_energy_payload, building_monthly_payload, ngToast, gb_req_info, all_columns,
```

Comment:
Will check.

On Tue, Mar 29, 2016 at 12:57 PM, Nicholas Long <notifications@github.com>
wrote:

> In seed/static/seed/js/controllers/building_detail_controller.js
> <https://github.com/SEED-platform/seed/pull/863#discussion_r57758952>:
>
> > @@ -37,7 +38,7 @@ angular.module('BE.seed.controller.building_detail', ['ngToast'])
> >    'label_helper_service',
> >    'default_columns',
> >    function($scope, $routeParams, $uibModal, $log, building_services, project_service, building_payload,
> > -           building_finer_energy_payload, building_monthly_payload, gb_req_info, all_columns,
> > +           building_finer_energy_payload, building_monthly_payload, ngToast, gb_req_info, all_columns,
>
> there appears to be some formatting issues here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/863/files/07fcd5c43cfd1d50ca513fd8bd38d7410b999749#r57758952>
>